API.md

```diff
@@ -155,6 +155,15 @@ Sends a message to all the subscribed clients where:
 - `message` - the message sent to the clients. Can be any type which can be safely converted to
   string using `JSON.stringify()`.
 
+
+### `server.eachSocket(callback, options)`
+
+Iterates over all connected sockets, optionally filtering on those that have subscribed to
+a given subscription. This operation is synchronous.
+- `callback` - Iteration callback in the form `function(socket)`.
```

Comment:
But not the hapi style guide. callback and next have specific meanings, and are always only called once.